src/words/ast/INodeQueueSay.java

```diff
@@ -31,6 +32,9 @@ public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException
 		} else {
 			object = environment.getObject(identifier.stringValue);
 		}
+		if (object == null) {
+			throw new WordsObjectNotFoundException(identifier.stringValue);
```

Comment:
If property is of type WordsProperty.PropertyType.OBJECT, then property.objProperty cannot be null.

I agree this check can be removed (and in INodeQueueMove).  I think it's legacy code from when environment.getObject() returned null when an object wasn't found instead of throwing an exception.  We changed it to throw an exception but didn't change it here.